### ISSUE TITLE
[Bug] Parser importing fails

### ISSUE THREAD

**Issue by savindi-wijenayaka** [manchenkoff/openapi3-parser#22]
## Problem
When trying to import the library to the code, it gives `TypeError: 'type' object is not subscriptable` 

## Steps to reproduce

1. Create new Conda environment. (`conda create -n test python==3.8 pip`)
2. Create a new file and add `from openapi_parser import parse`
3. Run file

## Stack trace
```
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from openapi_parser import parse
  File "/home/savindi/anaconda3/envs/test/lib/python3.8/site-packages/openapi_parser/__init__.py", line 7, in <module>
    from .parser import parse
  File "/home/savindi/anaconda3/envs/test/lib/python3.8/site-packages/openapi_parser/parser.py", line 3, in <module>
    from .builders import *
  File "/home/savindi/anaconda3/envs/test/lib/python3.8/site-packages/openapi_parser/builders/__init__.py", line 1, in <module>
    from .info import InfoBuilder
  File "/home/savindi/anaconda3/envs/test/lib/python3.8/site-packages/openapi_parser/builders/info.py", line 4, in <module>
    from ..specification import Contact, Info, License
  File "/home/savindi/anaconda3/envs/test/lib/python3.8/site-packages/openapi_parser/specification.py", line 44, in <module>
    class Schema:
  File "/home/savindi/anaconda3/envs/test/lib/python3.8/site-packages/openapi_parser/specification.py", line 47, in Schema
    enum: Optional[list[Any]] = None
TypeError: 'type' object is not subscriptable
```
## Affected Versions
openapi3-parser==1.1.1

## Environment
Ubuntu 20.04
Python 3.8

**Comment by manchenkoff** [comment]
@savindi-wijenayaka 
Hi, I will try to find the problem in a few days. Did you try to use Python 3.9?

**Comment by senarijit2021** [comment]
Same issue I am facing any updates on this??

**Comment by manchenkoff** [comment]
Hey @senarijit2021 👋 as you can see [here](https://github.com/manchenkoff/openapi3-parser/blob/master/Pipfile#L17), current package requires Python 3.9, please try with this version, if it doesn't work, feel free to tag me again

**Comment by manchenkoff** [comment]
closed because version 3.9 solves this issue
